browser/css/btns.css

```diff
@@ -132,6 +132,7 @@ button.jsdialog img {
 
 .ui-button-box .jsdialog.ui-pushbutton:not(.hidden) {
 	display: initial !important;
+	font-size: 12px;
```

Comment:
don't we have variable for that?

Reply:
@eszkadev resolved this to use --default-font-size from cool.css